Integrations/integration-ArcSightLogger.yml

```diff
@@ -146,20 +146,17 @@ script:
         var title = 'ArcSight Logger - Events';
         var context = events;
         entry.HumanReadable = tableToMarkdown(title, events);
-        entry.EntryContext = {
-            ArcSight: {
-                Events: context
-            }
-        };
+        entry.EntryContext = {};
+        entry.EntryContext['ArcSightLogger.Events(val.rowId === obj.rowId)'] = context;
```

Comment:
rawId is not good enough I think. Because the row is relative per search.
So for each search there will be rawId = 0, but the events could be different